src/generators/app/templates/src/shared/api/index.js

```diff
@@ -1,7 +1,18 @@
-import config from 'config';
-import fixtures from 'api/fixtures';
+/* eslint no-confusing-arrow: 0 */
+
+import { reduce } from 'lodash';
+import fixtures from 'api/http/fixtures';
 import http from 'api/http';
 
+const ApiMiddleware = (module, moduleName) => reduce(module, (result, value, key) => (
+  {
+    ...result,
+    [key]: (data = {}, options = {}) =>
+      options.fixture ? fixtures[moduleName][key](data, options) : value(data, options)
+  }
+), {});
+
+
```

Comment:
Ok I see what you're going for here, I like the approach, since it is opt-in per call!

I want to clarify this code a bit with more explicit variable names. I think it would also be great to get more granular with the fixtures, so we can have multiple fixtures for the same API call.